### DISCUSSION TITLE
sim.plot1D?

### DISCUSSION THREAD

**Discussion by dalarev** [NanoComp/meep#1924] [Q&A]
Is there a way to visualize the setup/geometry of 1D simulations?

I understand an assumption would have to be made about the size of the 2nd dimension, but it seems like a useful feature. 

A function could be created to add an arbitrary length to the problem setup for visualization with sim.plot2D, but I'm wondering if there is a less cumbersome method.

Thanks!

edit: There is a nice way to do this here: https://www.learnelectricalengineering.com/computational_em/lesson2_11.php

**Comment by smartalecH** [comment]
Currently there's no API for what you're describing. Care to submit a PR?

**Reply by dalarev** [reply]
Yes! I'll work on polishing the code (and learn to properly use Github) to submit a PR.
